### DISCUSSION TITLE
Is it possible to add a few lines to save the subtitles from a all vids in a playlist?

### DISCUSSION THREAD

**Discussion by serenitytomorrow** [TeamNewPipe/NewPipe#10149] [Questions] [answered]
I don't program in Java but isn't it easy to implement? All you have to do is save the stream in a file.

**Comment by Stypox** [comment, marked as answer]
This is not a question but rather a feature request, unless you are asking for help on how to implement that. Also, even if it was simple to implement (which it isn't, because it would require setting up a service to perform things in the background even if the app is closed), it doesn't fit into the current NewPipe downloader structure (but it will hopefully fit in the future). Anyway, duplicate of #5482.
